mmcls/datasets/samplers/__init__.py

```diff
@@ -1,4 +1,6 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .builder import SAMPLERS
 from .distributed_sampler import DistributedSampler
+from .repeat_aug import RepeatAugSampler
 
-__all__ = ['DistributedSampler']
+__all__ = ('DistributedSampler', 'RepeatAugSampler', 'SAMPLERS')
```

Comment:
Better to use list, refers to [PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names)